instrumentation/github.com/aws/aws-lambda-go/otellambda/wrapLambdaHandler.go

```diff
@@ -157,7 +158,8 @@ func (whf *wrappedHandlerFunction) wrapper(handlerFunc interface{}) func(ctx con
 		defer whf.instrumentor.tracingEnd(ctx, span)
 
 		if len(eventJSON) > 0 {
-			span.SetAttributes(attribute.KeyValue{Key: "faas.event", Value: attribute.StringValue(string(eventJSON))})
+			faasEventAttribute := obfuscator.ObfuscateAttributeValue(attribute.KeyValue{Key: "faas.event", Value: attribute.StringValue(string(eventJSON))})
```

Comment:
What about faas.res? There are two attributes

Reply:
we don't have the faas.res in the obfuscation keys

Reply:
It should be there

Reply:
and why you added  only the faas.event to the attributes?